### PR TITLE
Phase 2

### PR DESCRIPTION
**Pull request by AmirAbrams**
### What is the purpose of this pull request (PR)?
- Add stubs for these fluid operations:
`OP_SWAP_SOVEREIGN_ADDRESS`, `OP_UPDATE_FEES`, `OP_FREEZE_ADDRESS`, `OP_RELEASE_ADDRESS`
- Fully functional `OP_MINT`, `OP_REWARD_DYNODE` and `OP_REWARD_MINING` fluid operation codes
#### Was this PR tested and how? (If testing is not needed, please explain why)
Yes, using Ubuntu 64bit.

dynamic-fluid-dynode_test4_20171208

dynamic-fluid-mem-pool-check2_20171208

dynamic-fluid-dynode_getfluidhistory_command_20171209

### CODE REVIEW

**Comment by ghost** [comment]
OP_SWAP_SOVEREIGN_ADDRESS not needed as we depend on Identities

**Comment by AmirAbrams** [comment]
@plaxton, maybe we should change the operation code to `OP_SWAP_SOVEREIGN_IDENTITY`
We need to be able to revoke and replace sovereign keys.

File: src/chainparams.cpp [hunk 1]
```diff
@@ -202,12 +203,12 @@ class CMainParams : public CChainParams {
 
         nPoolMaxTransactions = 3;
         nFulfilledRequestExpireTime = 60 * 60; // fulfilled requests expire in 1 hour
-        strSporkPubKey = "048459e70138ccb109b38de57aa87b5b49b1b7c92550ee29b25e8eca195063d8872cc256bb35688ff6159112b802989f4b87a87d5ee1d6c1747c6e4bcdb68a3dae";
+        strSporkPubKey = "";
```
Comment (AmirAbrams) [inline src/chainparams.cpp]: We keep deleting the spork pub key and if we don't have one, we will not be able to spork.  Same with the alert pub key.

File: src/chainparamsseeds.h [hunk 2]
```diff
@@ -19,6 +19,7 @@
 //	188.166.173.136:32300
 //	159.203.218.28:32300
 static SeedSpec6 pnSeed6_main[] = {
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x55,0xd6,0x5f,0xb1}, 32300}
 };
```
Comment (AmirAbrams) [inline src/chainparamsseeds.h]: We should remove this old seed.

**Comment by ghost** [comment]
@AmirAbrams, if you consider that the sovereign control is with the identities, they can be changed at will, marking that opcode redundant

**Comment by spencerlievens** [comment]
I guess this only leaves warnings to fix :P

**Comment by spencerlievens** [comment]
ACK

**Review by spencerlievens — APPROVED** [review]
ACK

**Pull request merged**